src/hooks/item.test.ts

```diff
@@ -58,25 +59,20 @@ describe('Items Hooks', () => {
   });
 
   describe('useOwnItems', () => {
-    const route = `/${GET_OWN_ITEMS_ROUTE}`;
-    const hook = () => hooks.useOwnItems();
+    const route = `/${buildGetOwnItemsRoute({ page: 1, name: '' })}`;
+    const hook = () => hooks.useOwnItems({ page: 1, name: '' });
 
     it(`Receive own items`, async () => {
       const response = ITEMS;
       const endpoints = [{ route, response }];
       const { data } = await mockHook({ endpoints, hook, wrapper });
-
-      expect(data).toEqualImmutable(response);
+      expect(JSON.stringify(data)).toEqual(JSON.stringify(response));
```

Comment:
We've also noticed a bug with `toEqualImmutable`, we've changed them to `expect(Immutable.is(data,response)).toBeTruthy();`